discovery-infra/test_infra/config/_cluster_config.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass
+
+from test_infra.config import EnvConfig
+from test_infra.helper_classes.cluster_config import ClusterConfig, ClusterInstallConfig
+
+
+@dataclass
+class DefaultInstallConfig(ClusterInstallConfig):
+
+    def __post_init__(self):
+        self.iso_download_path = self.iso_download_path \
+            if self.iso_download_path is not None else EnvConfig.get("iso_download_path")
```

Comment:
maybe ``self.iso_download_path = self.iso_download_path or EnvConfig.get("iso_download_path")``?

Reply:
For boolean values it might not work